corehq/apps/api/resources/v0_5.py

```diff
@@ -935,3 +935,48 @@ def prepend_urls(self):
             url(r"^(?P<resource_name>%s)/(?P<case_type>[\w\d_.-]+)/$" % self._meta.resource_name,
                 self.wrap_view('dispatch_list'))
         ]
+
+ODATA_XFORM_INSTANCE_RESOURCE_NAME = 'Forms'
+
+
+class ODataXFormInstanceResource(v0_4.XFormInstanceResource):
+
+    xmlns = None
+
+    def dispatch(self, request_type, request, **kwargs):
+        if not toggles.ODATA.enabled_for_request(request):
+            raise ImmediateHttpResponse(response=HttpResponseNotFound('Feature flag not enabled.'))
+        self.app_id = kwargs['app_id']
+        self.xmlns = kwargs['xmlns']
```

Comment:
are these always going to be set as `kwargs`? otherwise this will throw a key error

Reply:
Yes, the keyword is required:
https://github.com/dimagi/commcare-hq/pull/24481/files/a4b5742ac7b937990edff2df1537ff609e7d7d4a..a1aee451ddc634db0da4c1dca7f1077662c22568#diff-12c6a125d25a13a47685017218f915a2R980